CMakeLists.txt

```diff
@@ -469,6 +493,15 @@ if(ALLEGRO_UNIX) # not MACOSX
         endif(ALSA_FOUND)
     endif(WANT_ALSA)
 
+    pkg_check_modules(SDL2 sdl2)
```

Comment:
Yes I know. This is part of edward rudd's linux port.

Reply:
This has been hiding in the macos port for a while (because it was originally based on rudd's linux fixes) but I tested it and it works pretty well! Even in a VM!